packages/devtools_app/lib/src/utils.dart

```diff
@@ -71,6 +71,8 @@ String prettyPrintBytes(
   int mbFractionDigits = 1,
   bool includeUnit = false,
 }) {
+  // Ensure a small number of bytes does not print as 0 KB.
+  // If bytes >= 52 and kbFractionDigits == 1, it will start rounding to 0.01 KB.
```

Comment:
typo: i think you mean 0.1 here?

On second thought, this might be a little too specific this is dependent upon the value of `kbFractionDigits`. Add a TODO to generalize this to the number of digits specified. It's fine to land this CL and fix later.

Reply:
and for now add a check that kbFractionDigits == 1 to the if check `if (bytes.abs() < 52)`